pkg/local_object_storage/metabase/metadata.go

```diff
@@ -422,3 +422,26 @@ func (x *metaAttributeSeeker) Get(id []byte, attr string) (attributeValue []byte
 	}
 	return key[len(pref):], nil
 }
+
+// returns both zero if object is missing.
+func resolveContainerByOID(tx *bbolt.Tx, metaOIDKey []byte) (cid.ID, error) {
```

Comment:
Looks like it never returns non-nil error.

Reply:
only if blindly believe that `tx.ForEach(f)` returns `f` errors only. This is not stated in docs

Reply:
OK, it'd be a DB error in which case failing completely makes sense.